client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -608,6 +608,7 @@ const PatternAssembler = ( {
 		<StepContainer
 			className="pattern-assembler__sidebar-revamp"
 			stepName="pattern-assembler"
+			stepSectionName={ currentScreen.name }
```

Comment:
@arthur791004 this leads to `calypso_signup_previous_step_button_click` not being tracked for this step when the back button is clicked. 

I'm working out on some refactors for the Tracks events lodding in Stepper. I need to get my head around this need (to not log the event in Tracks). Just some clarity/quick questions to get my head around for now:

1. Is the `pattern-assembler` step particular to a single flow, or a generalized step that other flows can utilize? 
2. When would `currentScreen.name` be falsy in this step?